README.md

```diff
@@ -4,7 +4,7 @@
 
 _Hagall means Hail in Old Norse._
 
-Hagall is a Real-Time Networking Server responsible for processing, responding to and broadcasting networking messages to connected clients (participants) in a session similar to how a multiplayer networking engine handles message passing in a first-person-shooter game.
+Hagall, also known as Relay Server, is a Real-Time Networking Server responsible for processing, responding to and broadcasting networking messages to connected clients (participants) in a session similar to how a multiplayer networking engine handles message passing in a first-person-shooter game.
```

Comment:
I guess this should be "Relay, previously known as Hagall, is a Real-Time Networking Server"